docker/api/service.py

```diff
@@ -306,7 +306,7 @@ def tasks(self, filters=None):
     def update_service(self, service, version, task_template=None, name=None,
                        labels=None, mode=None, update_config=None,
                        networks=None, endpoint_config=None,
-                       endpoint_spec=None):
+                       endpoint_spec=None, use_current_spec=False):
```

Comment:
Can we rename this to `fetch_current_spec`, so it's clearer that this will result in an additional roundtrip to retrieve the spec?